src/Cynthia.Card/src/Cynthia.Card.Common/CardEffects/DIY/Skellige/Copper/DrummondPillager.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Alsein.Extensions;
+
+namespace Cynthia.Card
+{
+    [CardEffectId("70081")] //德拉蒙德-皮勒格
+    public class DrummondPillager : CardEffect, IHandlesEvent<AfterCardDiscard>
+    {
+        //when discarded, strengthen all drummond units by 1 wherever they are.
+        public DrummondPillager(GameCard card) : base(card)
+        {
+        }
+
+        private async Task StrengthenAllDrummond(int Strengthen)
+        {
+            var targets = Game.PlayersHandCard[PlayerIndex].Concat(Game.GetPlaceCards(PlayerIndex)).Concat(Game.PlayersDeck[PlayerIndex]).FilterCards(filter: x => x.HasAllCategorie(Categorie.ClanDrummond) && x != Card);
+
+            foreach (var target in targets)
+            {
+                await target.Effect.Strengthen(1, Card);
+            }
+        }
+
+        private const int Strengthen = 1;
```

Comment:
```suggestion
```
looks like this code is useless? Also maybe use `StrengthenPoint` if you still want to keep this code, as `Strengthen` is already used as a method name.

Reply:
> I read the code carefully and basically, it looks good to me.
> 
> But I still strongly recommend you run it locally to ensure that there are no bugs and that everything works as intended. (Though if you strongly request we can merge it directly.)
> 
> Please let me know when you are ready to merge and we can ask @DeusSeuca to merge the code.

Thanks. Since I'm a begginer and don't want to cause bugs I'm trying to make it work locally, I'm also looking into your other comments, will tell you when it's ready.

Reply:
So I commited the change and it broke the code, I will try your second solution